contracts/LoanManager.sol

```diff
@@ -271,15 +271,14 @@ contract LoanManager is Restricted, TokenReceiver {
     function calculateLoanValues(LoanProduct storage product, uint repaymentAmount)
     internal view returns (uint loanAmount, uint interestAmount) {
         // calculate loan values based on repayment amount
-        loanAmount = repaymentAmount.mul(product.discountRate).div(1000000);
+        loanAmount = repaymentAmount.mul(product.discountRate).div(1e6);
         interestAmount = loanAmount > repaymentAmount ? 0 : repaymentAmount.sub(loanAmount);
     }
 
     // the token/ETH rate of the margin, under which the loan can be "margin called" (collected)
     function calculateMarginCallRate(uint32 minCollateralRatio, uint repaymentAmount, uint collateralAmount)
     internal pure returns (uint) {
-        // TODO: think about proper div rounding to use here
-        return uint(minCollateralRatio).mul(repaymentAmount).mul(1000000000000).div(collateralAmount);
+        return uint(minCollateralRatio).mul(repaymentAmount).mul(1e12).div(collateralAmount);
```

Comment:
the constant expression `1e18 / 1e6' might be more explanatory than `1e12` (and should be calculated at compile time, so no runtime cost)